### ISSUE TITLE
Deprecated: 'which' should be 'command -v'

### ISSUE THREAD

**Issue by WSLUser** [WhitewaterFoundry/Pengwin#713]
**Describe the bug**
When opening Pengwin, I see

``/usr/bin/which: this version of `which' is deprecated; use `command -v' in scripts instead.``

**To Reproduce**
Simply open Pengwin Dev.

**Expected behavior**
Use `command -v` instead.
**Screenshots**

**Additional context**
Everything using `which` will break unless the command is updated.
**Basic Troubleshooting Checklist**

[ ] I have searched Google for the error message.
[ ] I have checked official WSL troubleshooting documentation: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#confirm-wsl-is-enabled.
[ ] I have searched the official Microsoft WSL issues page: https://github.com/Microsoft/WSL/issues.
[x] I have searched the Pengwin issues page: https://github.com/WhitewaterFoundry/Pengwin/issues.
[ ] I have run Windows 10 updates and restarted.
[ ] I have restarted WSL service: `wsl.exe --terminate WLinux` `wsl.exe --shutdown`.

What other troubleshooting have you attempted?

Insert here:

**Pengwin Version**

Run `grep "PENGWIN_VERSION" /etc/os-release` in Pengwin.

Insert here: `PENGWIN_VERSION="21.10.1"`

**WSL general info**

Run `wslsys` in Pengwin.

Insert here: 

```
Release Install Date: Tue Jun  8 15:34:57 UTC 2021
Branch: vb_release
Build: 19043
Full Build: 19041.1.amd64fre.vb_release.191206-1406
Display Scaling: 2
Locale (Windows): en_US
Theme (Windows): dark
Uptime (Windows): 28d 0h 44m
Version (WSL): 2
Uptime (WSL): 0d 0h 8m
Release: Pengwin
Kernel: Linux 5.14.6-microsoft-standard-WSL2
Packages: 1810
```

**Comment by WSLUser** [comment]
@crramirez so this gets looked as soon as possible. Should simply be a matter of find/replace for all scripts. Probably can use something like `awk` to make it easier to replace all uses at once.

**Comment by crramirez** [comment]
Hello @WSLUser,

Thanks for reporting. We have changed which by command -v in many places because shellcheck alert it. But I have never though that which will be deprecated.

Regards,
Carlos

**Comment by WSLUser** [comment]
Yeah I never thought it would be either but here we are. That is such as useful command. When support is removed completely, I'd probably create an alias 'which' to use instead. Probably good idea to add that alias to a user's `.profile` in Pengwin and FedoraRemix.
